### PR TITLE
PCM-7506 - Remove escalation flag while creating escalation from PCM .

### PR DESCRIPTION
**Pull request by anujsi**
@gandhikeyur @vrathee Please review.

### CODE REVIEW

**Comment by anujsi** [comment]
@vrathee I am merging this PR please let me know if you have any suggestion

File: app/cases/controllers/requestManagementEscalationModal.js [hunk 1]
```diff
@@ -129,17 +123,10 @@ export default class RequestManagementEscalationModal {
...
-                    CaseService.checkForCaseStatusToggleOnAttachOrComment();
-                    if (escalationNum !== undefined) {
-                        AlertService.addSuccessMessage(gettextCatalog.getString('Your Escalation request has been sent successfully'));
-                    }
-                },
-                function (error) {
-                    $scope.showErrorMessage(error);
-                });
+                CaseService.checkForCaseStatusToggleOnAttachOrComment();
+                if (escalationNum !== undefined) {
```
Comment (pranvikasha) [inline app/cases/controllers/requestManagementEscalationModal.js]: @anuj I don't see escalationNum being populated after backend call is made.. How yould you get the new escalation number and show this message?

File: app/cases/controllers/showRMEEscalation.js [hunk 2]
```diff
@@ -84,19 +84,12 @@ export default class ShowRmeEscalation {
...
-                        AlertService.addSuccessMessage(gettextCatalog.getString(alertMessage));
-                        $scope.submittingRequest = false;
-                    }
-                },
-                function (error) {
-                    $scope.showErrorMessage(error);
-                });
+                CaseService.checkForCaseStatusToggleOnAttachOrComment();
+                if (escalationNum !== undefined) {
+                    AlertService.clearAlerts();
```
Comment (pranvikasha) [inline app/cases/controllers/showRMEEscalation.js]: same here

**Comment by pranvikasha** [comment]
Reviewed!

**Pull request merged**